package.json

```diff
@@ -44,7 +44,7 @@
     "lodash": "^3.10.0",
     "method-override": "^2.3.3",
     "mocha": "~1.20.0",
-    "mongoose": "^4.0.6",
+    "mongoose": "4.1.11",
```

Comment:
Could have been in there. Their commit history is funky to read with the merging of 3.8.x into master between the 4.1.11 and 4.1.12 releases. I'll play and see if I can break it.